webpack.config.js

```diff
@@ -153,7 +159,13 @@ module.exports = function (webpackEnv) {
         new webpack.optimize.CommonsChunkPlugin({name: 'vendor'}),
         new webpack.LoaderOptionsPlugin({
           options: {
-            postcss: () => [autoprefixer]
+            postcss: () => [autoprefixer],
+            worker: {
+              output: {
+                filename: 'hash.worker.js',
```

Comment:
These lines allow for require statements in the form of: 

```
const Worker = require('worker-loader![RELATIVE FILE PATH HERE]');
```

To be loaded into their own worker.js file, which a hash id for cache busting on the client.

This is required since [Web workers must be created from a URI](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Using_web_workers#Spawning_a_dedicated_worker) And cannot be "officially" loaded in the same JS file. [There is a hack for this](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Using_web_workers#Embedded_workers), but I would prefer to have webpack handle this, than us try to inline the JS ourselves into a script tag